src/modules/admissions/components/AdmissionDashboard/AdmissionsShortcutPanel.tsx

```diff
@@ -38,6 +39,13 @@ const shortcuts = [
     color: 'samfundet-red',
     link: '/admissions/todays-interviews',
   },
+  {
+    title: 'Intervjuoversikt',
+    icon: IconAbacus,
+    color: 'samfundet-red',
```

Comment:
Making a task on removing usage of the main color "samfundet-red", because this is primaryColor and should be referred to instead